File: SecondYearCS/OOP/Civitas/src/civitas/OperacionInmobiliaria.java
package civitas;

public class OperacionInmobiliaria {
    private int numPropiedad;
    private GestionesInmobiliarias gestion;
    
    public OperacionInmobiliaria(GestionesInmobiliarias gest, int ip){
        this.numPropiedad = ip;
        this.gestion = gest;
    }
    
    public GestionesInmobiliarias getGestion(){
        return this.gestion;
    }
    
    public int GetNumPropiedad(){
        return this.numPropiedad;
    }
    
}

File: SecondYearCS/OOP/CivitasRB/lib/operacion_inmobiliaria.rb
module CivitasRB
  class OperacionInmobiliaria
    def initialize(gest,ip)
      @numPropiedad = ip
      @gestion = gest
    end
    
    def getGestion
      return @gestion
    end
    
    def getNumPropiedad
      return @numPropiedad
    end
  end
end
